server/chat-plugins/mafia.ts

```diff
@@ -2285,8 +2289,8 @@ export const commands: ChatCommands = {
 			game.ideaPick(user, args);
 		},
 
-		'!ideareroll': true,
 		ideareroll(target, room, user) {
+			if (!room) return this.requiresRoom();
```

Comment:
you deleted the global tag and then added a requiresRoom check to this and a bunch of other mafia (possibly other) commands, which is definitely wrong.